metrics/web/test/dashboard/presentation/models/dashboard_page_parameters_model_test.dart

```diff
@@ -0,0 +1,100 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:metrics/dashboard/presentation/models/dashboard_page_parameters_model.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("DashboardPageParametersModel", () {
+    const projectFilter = 'projectFilter';
+    const projectGroupId = 'projectGroupId';
+    const pageParametersMap = {
+      'projectFilter': projectFilter,
+      'projectGroupId': projectGroupId,
+    };
+    final pageParametersModel = DashboardPageParametersModel(
+      projectFilter: projectFilter,
+      projectGroupId: projectGroupId,
+    );
+
+    test(
+      "creates an instance with the given parameters",
+      () {
+        final pageParametersModel = DashboardPageParametersModel(
+          projectFilter: projectFilter,
+          projectGroupId: projectGroupId,
+        );
+
+        expect(pageParametersModel.projectFilter, equals(projectFilter));
+        expect(
+          pageParametersModel.projectGroupId,
+          equals(projectGroupId),
+        );
+      },
+    );
+
+    test(
+      ".copyWith() creates a new instance with the same fields if called without params",
+      () {
+        final copiedPageParametersModel = pageParametersModel.copyWith();
+
+        expect(copiedPageParametersModel, equals(pageParametersModel));
+      },
+    );
+
```

Comment:
✅ 
Would it be great to add a test case to cover the `props` property and check equality? Please consider other places where equitable classes are tested.